content/en/docs/create-project/build-packs.md

```diff
@@ -134,5 +134,28 @@ If your build pack requires custom logic to detect it then let us know
 and we can help patch [jx import](/commands/jx_import/) to work better for your build pack.
 For example, we have some custom logic for handling [Maven and Gradle better](https://github.com/jenkins-x/jx/blob/712d9edf5e55aafaadfb3e0ac57692bb44634b1c/pkg/jx/cmd/common_buildpacks.go#L82:L108).
 
+# Using custom build packs
```

Comment:
This is the wrong heading level. In fact, it is wrong across the document. You should not have more than one level one heading which in our case is actually given by the title in the metadata. So when it comes to Markdown your "highest" level is "##"

If you look at the page you can see how the headings for "Creating new build packs" and "Using custom build packs" are sticking out and are not linked in the right-hand navigation.